src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/MetaApi/Module.ts

```diff
@@ -6,5 +6,5 @@ export var moduleName = "adhMetaApi";
 export var register = (angular, metaApi) => {
     angular
         .module(moduleName, [])
-        .factory("adhMetaApi", () => new AdhMetaApi.Service(metaApi));
+        .provider("adhMetaApi", () => new AdhMetaApi.Service(metaApi));
```

Comment:
Why is `AdhMetaApi.Service(metaApi)` still named "service" even though it's registered as a provider?